client/src/components/BtnGoBack.jsx

```diff
@@ -9,7 +9,7 @@ function BtnGoBack() {
   };
   return (
     <button
-      aria-labelledby="button-back"
+    aria-label="Bouton retour"
```

Comment:
As it is meant to be read by accessibility tools, we should be more precise, like "Retour vers la page précédente" 
No need to mention the type of tag because the tool will read it